README.md

```diff
@@ -64,12 +64,11 @@ source /opt/ros/noetic/setup.bash
 ```
 3. Install additional dependencies
 ```
-sudo apt install python-rosinstall-generator python-catkin-tools
+sudo apt install python-rosinstall-generator python-catkin-tools python3-vcstool
```

Comment:
The `python-rosinstall-generator` for us also had to change to `python3-rosinstall-generator`.